op-program/host/cmd/main.go

```diff
@@ -21,7 +21,11 @@ var (
 var VersionWithMeta = func() string {
 	v := version.Version
 	if GitCommit != "" {
```

Comment:
This seems like it could be a standard function in `op-service` to dedupe code?

Reply:
We could do that as a follow up, but I'm keen to get this in so the devnet works again.  Apparently an unresolved conversation drops the PR out of the merge queue again. :)